sw/device/tests/crypto/cryptotest/firmware/sha3_sca.c

```diff
@@ -550,20 +597,27 @@ status_t handle_sha3_sca_seed_lfsr(ujson_t *uj) {
  * @param uj The received uJSON data.
  */
 status_t handle_sha3_sca_init(ujson_t *uj) {
+  // Read mode. FPGA or discrete.
+  /*cryptotest_sha3_sca_fpga_mode_t uj_data;
+  TRY(ujson_deserialize_cryptotest_sha3_sca_fpga_mode_t(uj, &uj_data));
+  if ((*uj_data.fpga_mode & 0xFF) == 0x01) {
+    fpga_mode = true;
+  }*/
+  fpga_mode = false;
+
```

Comment:
Is this commented out intentionally?

Reply:
Fixed, commenting this out is a bug that resulted from some experiments I did.